app/_locales/en/messages.json

```diff
@@ -1035,6 +1035,21 @@
     "message": "File import not working? Click here!",
     "description": "Helps user import their account from a JSON file"
   },
+  "flaskExperimentalText1": {
+    "message": "Using Flask can greatly increase your risk of fund loss:"
+  },
+  "flaskExperimentalText2": {
+    "message": "if you use it to install non-trustworthy Snaps"
+  },
+  "flaskExperimentalText3": {
+    "message": "if you do not review confirmations before approving changes"
+  },
+  "flaskExperimentalText4": {
+    "message": "if you interact with unfamiliar smart contracts"
```

Comment:
Not sure how these messages are being display but do we want to capitalize the first letters?
"If you use it to install non-trustworthy Snaps"

Reply:
Ah just saw the design ignore ^